js/admin.js

```diff
@@ -536,7 +536,12 @@ function showRedirectProductOptions(show)
 function redirectSelectChange()
 {
   redirectTypeValue = $('#redirect_type :selected').val();
-  if (redirectTypeValue == '404' || redirectTypeValue == '410' )
+  if (redirectTypeValue == '404' || 
+      redirectTypeValue == '410' || 
+      redirectTypeValue == 'default' || 
+      redirectTypeValue == '200-displayed' || 
+      redirectTypeValue == '404-displayed' || 
+      redirectTypeValue == '410-displayed')
```

Comment:
I see these used in couple place already, can't we use some constants or smth, to avoid repeating and more important - avoid having magic strings|numbers?
Though, Im not the expert of js, especially in this old code part, maybe @NeOMakinG can elaborate?